frontend/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    println!("cargo:rerun-if-changed=../proto/helloworld.proto");
```

Comment:
```suggestion
    println!("cargo:rerun-if-changed=../proto");
```
We can watch the whole directory